modules/math/shared/src/main/scala/gsp/math/RightAscencionAngularVelocity.scala

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gsp.math
+
+import cats._
+import coulomb._
+import coulomb.cats.implicits._
+import gsp.math.units._
+import gsp.math.optics.SplitMono
+import monocle.Iso
+import spire.math.Rational
+import spire.std.long._
+
+/**
+  * Representation of Right Ascension angular velocity in microarcseconds per year
+  * It is used to represent proper velocity in RA
+  */
+final case class RightAscensionAngularVelocity(μasy: Quantity[Long, MicroArcSecondPerYear]) {
```

Comment:
In the end `DeclinationAngularVelocity` is identical to `RightAscensionAngularVelocity`, though I agree we want to have distinct types for them.  One option would be to copy the pattern used for `Offset` and have a `AngularVelocity.Component` with an `Axis` type parameter (having `RA` and `Dec` options).

Reply:
That's an interesting approach. I'll take a look.
It would be nice to have the same logic as `Offset` even though they are different beasts